frontend/lib/src/components/widgets/Multiselect/Multiselect.tsx

```diff
@@ -359,7 +359,7 @@ class Multiselect extends React.PureComponent<Props, State> {
                         paddingLeft: theme.spacing.sm,
                         marginLeft: theme.spacing.none,
                         marginRight: theme.spacing.sm,
-                        height: "28px",
+                        height: `calc(${theme.sizes.minElementHeight} - 2 * ${theme.spacing.xs})`,
```

Comment:
Could you please leave a comment where this formula is coming from?

Reply:
Added 👍